cli/command/plugin/install.go

```diff
@@ -39,52 +43,86 @@ func newInstallCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags := cmd.Flags()
 	flags.BoolVar(&options.grantPerms, "grant-all-permissions", false, "Grant all permissions necessary to run the plugin")
 	flags.BoolVar(&options.disable, "disable", false, "Do not enable the plugin on install")
+	flags.StringVar(&options.alias, "alias", "", "Local name for plugin")
 
 	return cmd
 }
 
-func runInstall(dockerCli *command.DockerCli, opts pluginOptions) error {
-	named, err := reference.ParseNamed(opts.name) // FIXME: validate
+func getRepoIndex(ref distreference.Named) (*registrytypes.IndexInfo, error) {
```

Comment:
I feel like this function would be marginally less confusing if it took a string argument. Otherwise, it could be called something like `getRepoIndexFromUnnormalizedRef`.